src/main/java/io/opentracing/contrib/global/GlobalTracer.java

```diff
@@ -38,43 +38,26 @@
  * The {@linkplain GlobalTracer} will not attempt to choose between implementations:</li>
  * <li>If no single tracer service is found, the {@link io.opentracing.NoopTracer NoopTracer} will be used.</li>
  * </ol>
+ *
+ * @see io.opentracing.util.GlobalTracer
+ * @deprecated GlobalTracer is now part of OpenTracing-java
  */
-public final class GlobalTracer implements Tracer {
```

Comment:
Why do you go to great lengths to keep backwards compatibility by using reflection on getting the tracer set on the new implementation, but then break backwards compatibility by removing this interface?